src/Kovan-DssSpell.t.sol

```diff
@@ -1835,86 +2122,275 @@ contract DssSpellTest is DSTest, DSMath {
             bytes32(uint256(address(this)))
         );
 
-        ClipAbstract clipYFIA = ClipAbstract(addr.addr("MCD_CLIP_YFI_A"));
-        assertEq(clipYFIA.stopped(), 0);
-        clipMom.setBreaker(address(clipYFIA), 1, 0);
-        assertEq(clipYFIA.stopped(), 1);
-        clipMom.setBreaker(address(clipYFIA), 2, 0);
-        assertEq(clipYFIA.stopped(), 2);
-        clipMom.setBreaker(address(clipYFIA), 3, 0);
-        assertEq(clipYFIA.stopped(), 3);
-        clipMom.setBreaker(address(clipYFIA), 0, 0);
-        assertEq(clipYFIA.stopped(), 0);
+        ClipAbstract clipETHA = ClipAbstract(addr.addr("MCD_CLIP_ETH_A"));
+        assertEq(clipETHA.stopped(), 0);
+        clipMom.setBreaker(address(clipETHA), 1, 0);
+        assertEq(clipETHA.stopped(), 1);
```

Comment:
it might make sense to make sure that lo auctions can be triggered

Reply:
Yeah, same than above, didn't want to get the tests that complicated. It wasn't my intention to test that the ClipperMom works as intended but more that was correctly configured.